renderer.php

```diff
@@ -320,7 +320,7 @@ function local_envbar_title($match) {
         return '';
     }
 
-    $prefix = substr($match->showtext, 0, 4);
+    $prefix = addslashes(substr($match->showtext, 0, 4));
```

Comment:
You may instead want 

https://github.com/moodle/moodle/blob/master/lib/weblib.php#L117-L126

or `s()`

Reply:
Thanks Bredan. I have made changes to my code. Please check again.